backend/src/mail/views/blocks/member-button.njk

```diff
@@ -0,0 +1,5 @@
+<div class="button-div">
+  <a id="link-button" href="https://coreteam.gov.bc.ca/profile">Get Started</a>
```

Comment:
Given that there's a link to production here, can we ensure that mail from TEST / DEV are CLEARLY marked as such?